speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/ResearcherMenu.cpp

```diff
@@ -190,6 +190,75 @@ static void SetUserId(void*)
     GfuiEditboxSetString(s_scrHandle, m_userIdControl, m_userId);
 }
 
+/// @brief Saves the settings into the ResearcherMenu.xml file
+static void SaveSettingsToDisk() {
+    // Open file parameter
+    std::string strPath("data/menu/ResearcherMenu.xml");
+    char buf[512];
+    sprintf(buf, "%s%s", GfDataDir(), strPath.c_str());
+    void* readParam = GfParmReadFile(buf, GFPARM_RMODE_REREAD | GFPARM_RMODE_CREAT);
```

Comment:
Use some local directory instead (use for example things like GfParmWriteFileLocal in params.cpp or GfLocalDir() in tgf.h or maybe some other one (look for other local dir usages in menus))